### ISSUE TITLE
[Story interactive] Options are not buttons (a11y)

### ISSUE THREAD

**Issue by mszylkowski** [ampproject/amphtml#31587]
Currently the options are not marked as buttons in html so they are not accessible by tabbing in the page (when the interactive component is in the current page), and are not detected as buttons either by screen readers.

The options should be buttons instead of spans.
